src/routes/company_view.svelte

```diff
@@ -7,7 +7,6 @@
 	import { techexposure_company } from '../js/techexposure_company.js';
 	import { createErrorMessageDiv } from '../js/createErrorMessageDiv.js';
 	import * as d3 from 'd3';
-	import { union } from 'd3-array';
```

Comment:
Non-blocking: Does JS allow all data types to be `null`, or is it one of those languages that need to have like `union(null, int)` to have nullable values?

Reply:
I do not know!
@cjyetman ?

Reply:
I'm only removing this because `union` was imported but never actually used. 
But indeed, we may want to add it back again if we need it for what you suggest.

Reply:
🤷🏻

Reply:
don't know what this svelte script is for or how it works, but `union` from `d3.array` is used here:
https://github.com/RMI-PACTA/pacta-dashboard-svelte/blob/9a28e944718351db2d58ad2371932d93e2fa5959/src/js/techmix_sector.js#L130

Reply:
Importing `union` is already handled in that JS file: 
https://github.com/RMI-PACTA/pacta-dashboard-svelte/blob/9a28e944718351db2d58ad2371932d93e2fa5959/src/js/techmix_sector.js#L2